From 13f7fb3f5feb7d4e6129b262562db0955c5afe22 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 28 Feb 2012 17:42:06 +0100
Subject: [PATCH] view:
 s/elm_genlist_item_object_get/elm_object_item_widget_get/g

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/view/contact-view.c      |    2 +-
 src/view/message-list-view.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/view/contact-view.c b/src/view/contact-view.c
index b806a4f..d640b82 100644
--- a/src/view/contact-view.c
+++ b/src/view/contact-view.c
@@ -855,7 +855,7 @@ _field_edit_clicked(void *data, Evas_Object *obj, void *event_info)
 		_field_edit_add_edit_page(fd, content, _field_edit_fileselector_save_cb);
 	}
 	else {
-		edje_object_signal_emit((Evas_Object *) elm_genlist_item_object_get(fd->item), "start_edit", "elm");
+		edje_object_signal_emit((Evas_Object *) elm_object_item_widget_get(fd->item), "start_edit", "elm");
 		elm_entry_editable_set(fd->value_entry, EINA_TRUE);
 		elm_object_focus_set(fd->value_entry, EINA_FALSE);
 	}
diff --git a/src/view/message-list-view.c b/src/view/message-list-view.c
index 9bd51ff..69b3582 100644
--- a/src/view/message-list-view.c
+++ b/src/view/message-list-view.c
@@ -565,7 +565,7 @@ _contact_lookup(GError *error, GHashTable *contact, gpointer data)
 		message = (GHashTable *)elm_object_item_data_get(glit);
 		g_hash_table_insert(message, "Name",
 				      g_variant_ref_sink(g_variant_new_string(tmp)));
-		Evas_Object *obj = (Evas_Object *)elm_genlist_item_object_get(glit);
+		Evas_Object *obj = (Evas_Object *)elm_object_item_widget_get(glit);
 		edje_object_part_text_set(obj, "elm.name", tmp);
 		free(tmp);
 	}
-- 
1.7.8.5

